core/schema/module.go

```diff
@@ -1028,7 +1028,7 @@ func (s *moduleSchema) updateCodegenAndRuntime(ctx context.Context, mod *core.Mo
 				continue
 			}
 			gitAttrsContents = append(gitAttrsContents,
-				[]byte(fmt.Sprintf("/%s linguist-generated=true\n", fileName))...,
+				[]byte(fmt.Sprintf("/%s linguist-generated\n", fileName))...,
```

Comment:
@sipsma, I started by making the CLI create the `.gitattributes` file automatically and just looked into `linguist-generated` references in code, but then decided to simplify. In linguist docs `=true` isn't used so I'm commiting this change before it spreads in user modules.

To be clear, it works with it, just not how it's documented.